tests/test_calculator.hh

```diff
@@ -653,14 +673,20 @@ namespace rascal {
         result_idx += n_entries_per_key;
       }
       for (auto neigh : center) {
+        if (static_cast<size_t>(neigh.get_atom_tag()) >=
```

Comment:
can this be made into a function?  `this->structure_manager.is_ghost(neigh.get_atom_tag())` or something. It seems to rely on the underlying representation [real atoms ..., ghost atoms ...] which might change at some point.

Reply:
I literally left a comment on this a moment ago.

Reply:
I've seen the comment, I am just asking to implement this now rather than later :smiley: